acai-ios/SectionControllers/DetailViewSectionControllers/HeaderImageSectionController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  HeaderImageSectionController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/7/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class HeaderImageSectionController: ListSectionController {
+    
+    weak var delegate: dismissMenuItemViewController?
+    var menuItem: MenuItem!
+    
+    override init() {
+        super.init()
+    }
+    
+    override func numberOfItems() -> Int {
+        return 1
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let context = collectionContext, let option = menuItem else {
+            return .zero
+        }
+        let width = context.containerSize.width
+        return CGSize(width: width, height: 370)
+    }
+    
+    override func cellForItem(at index: Int) -> UICollectionViewCell {
+        let cell = collectionContext!.dequeueReusableCell(of: HeaderImageCollectionViewCell.self, for: self, at: index)
+        if let cell = cell as? HeaderImageCollectionViewCell {
```

Comment:
With IGListKit it's actually best practice to force-unwrap the cell, so you can get rid of the if let and have `let cell = collectionContext?.dequeueReusableCell(....) as! HeaderImageCollectionViewCell` instead.